Index: sonic-pi-2.7.0+dfsg1/Makefile
===================================================================
--- /dev/null
+++ sonic-pi-2.7.0+dfsg1/Makefile
@@ -0,0 +1,31 @@
+DESTDIR =
+
+QMAKE = qmake-qt4
+
+all: app/gui/qt/Makefile
+	$(MAKE) -C app/gui/qt
+
+app/gui/qt/ruby_help.h: app/gui/qt/ruby_help.tmpl
+	cp -f app/gui/qt/ruby_help.tmpl app/gui/qt/ruby_help.h
+	cd app/gui/qt && ../../server/bin/qt-doc.rb -o ruby_help.h
+
+app/gui/qt/Makefile: app/gui/qt/ruby_help.h app/gui/qt/SonicPi.pro
+	cd app/gui/qt && lrelease SonicPi.pro
+	cd app/gui/qt && $(QMAKE) -o Makefile SonicPi.pro
+
+clean:
+	#grabbed from app/gui/qt/rp-build-app
+	[ ! -f app/gui/qt/Makefile ] || $(MAKE) -C app/gui/qt distclean
+	rm -f app/gui/sonic-pi
+	rm -f app/gui/qt/lang/*.qm
+	rm -f app/gui/qt/book/*
+	rm -f app/gui/qt/help/*
+	rm -f app/gui/qt/info/*
+	rm -f app/gui/qt/help_files.qrc
+	rm -f app/gui/qt/ruby_help.h
+
+
+install:
+	$(MAKE) -C app/gui/qt install DESTDIR=$(DESTDIR) INSTALL_ROOT=$(DESTDIR)
+
+.PHONY: configure all clean install
