Index: sonic-pi/app/gui/qt/SonicPi.pro
===================================================================
--- sonic-pi.orig/app/gui/qt/SonicPi.pro	2015-12-05 09:33:03.264303500 +0000
+++ sonic-pi/app/gui/qt/SonicPi.pro	2015-12-05 09:33:03.260303477 +0000
@@ -110,6 +110,15 @@
 ICON = images/app.icns
 LIBS         += -lqscintilla2
 
+!win32 {
+	QMAKE_COPY = install --preserve-timestamps
+	QMAKE_STRIP = echo # Leave strip decision to package system
+
+	target.files = sonic-pi
+	target.path = /usr/bin
+	INSTALLS += target
+}
+
 win32 {
 	install_qsci.files = $$[QT_INSTALL_LIBS]\qscintilla2.dll
 	install_qsci.path = release
